include/maliput/api/maliput_design.h

```diff
@@ -160,8 +160,62 @@
 ///
 /// @subsubsection scale_length_and_tolerances Scale Length and Tolerances
 ///
-/// > TODO: Explain the concepts of linear tolerance, angular tolerance, and
-/// > characteristic scale length.
+/// The set of abstractions conformed by the `RoadGeometry`, `Junction`, `Segment`,
+/// `Lane` and `BranchPoint` concentrate various aspects of map data, among which we can
+/// identify and highlight:
+/// - Network topology.
+/// - Volume and surface information.
+/// - Dimensions.
+///
+/// Depending on the data source type that will feed an actual implementation of `maliput`
+/// these three aspects will be given, derived or found at the moment of parsing the information
+/// and building the entities. In particular, there is one important characteristic of the
+/// `RoadGeometry` abstraction: the @f$ G^1 @f$ continuity of the road surface. That implies:
+/// - The road surface is continuous.
+/// - The road surface is orientable.
```

Comment:
Can you unpack what you mean by 'orientable' here?

Reply:
Ah I see you have more details farther down. Maybe just a comment in brackets here to hint to the reader to keep reading).

Reply:
Removed the listing in favor of the previous section. [33d3c92](https://github.com/maliput/maliput/pull/558/commits/33d3c9201b5ff85b07e0692e5670ce42530c5c97)